omod/src/main/webapp/pages/summarydashboard/programEnrollments.gsp

```diff
@@ -0,0 +1,91 @@
+<%
+    ui.decorateWith("appui", "standardEmrPage")
+    ui.includeCss("coreapps", "clinicianfacing/patient.css")
+%>
+
+<script type="text/javascript">
+	var breadcrumbs = [
+		{ icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm' },
+		{ label: "${ ui.message(app.label)}"}
+	];
+
+	jq(function(){
+		jq(".tabs").tabs();
+		// make sure we reload the page if the location is changes; this custom event is emitted by the location selector in the header
+		jq(document).on('sessionLocationChanged', function() {
+			window.location.reload();
+		});
+	});
+</script>
+
+<div class="clear"></div>
+<div class="container">
+	<div class="dashboard clear">
+		<!-- only show the title div if a title has been defined in the messages.properties -->
+		<% if (ui.message(app.id + ".custom.title") != app.id + ".custom.title") { %>
+		<div class="title">
+			<h3>${ ui.message(app.id + ".custom.title") }</h3>
+		</div>
+		<% } %>
+		<div class="info-container column">
+			<% if (firstColumnFragments) {
+				firstColumnFragments.each {
+					// create a base map from the fragmentConfig if it exists, otherwise just create an empty map
+					def configs = [:];
+					if(it.extensionParams.fragmentConfig != null){
+						configs = it.extensionParams.fragmentConfig;
+					}
+					configs << [  app: it.appId ]
+			%>
+				${ ui.includeFragment(it.extensionParams.provider, it.extensionParams.fragment, configs)}
+			<%  }
+			} %>
+
+		</div>
+		<div class="info-container column">
+		</div>
+	</div>
+</div>
+<br/>
+<div class="dashboard info-header clear">
+	<h3>${ ui.message("coreapps.currently.enrolled") }:</h3>
+</div>
+<div>
+	<table id="enrollmentsTable" style="width: 100%">
+		<thead>
+			<tr>
+				<th>${ ui.message("Patient.name") }</th>
+				<th>${ ui.message("coreapps.emr.id") }</th>
+				<th>${ ui.message("coreapps.enrollment.startDate") }</th>
+				<th>${ ui.message("coreapps.treatment.state") }</th>
```

Comment:
This likely shouldn't be hardcoded to "Treatment State", as that is a specific workflow.  If we are displaying *all* current states in a single column, we'd need to name this something like "State(s)", but I agree with @mseaton 's comment below that it would likely be better to be column-per-workflow.